.github/workflows/check-links.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
         id: lc2
         uses: peter-evans/link-checker@v1
         with:
-          args: -d site/ -r site/ -x "(github.com/weaveworks/ignite($|/issue|/pull|/releases|/commit/|/releases)|squidfunk.github.io|ignite.readthedocs.io)"
+          args: -d site/ -r site/ -x "(github.com/weaveworks/ignite($|/issue|/pull|/releases|/commit/|/releases|/tree/master/pkg/apis)|squidfunk.github.io|ignite.readthedocs.io)"
```

Comment:
Checking `/tree/master` actually caught a valid error in the docs.
See commit: c53a6a4

Fails on master because we did not remove v1alpha1 API docs when we deprecated the API

Solution to the valid link-checking error was PR: #625

Reply:
So should I close this PR? The theme update is not strictly necessary.

Reply:
@dholbach hi, we had a discussion about this in the weekly meeting. We agreed that it's useful to run the link checks on the master branch only and we should disable the checks on the PR.
Can you update this PR to keep the theme update and update the github workflow for check-links to not run on pull requests?

Reply:
Merged #656